proxy/README.md

```diff
@@ -0,0 +1,113 @@
+# `cc-proxy`
+
+`cc-proxy` is a daemon offering access to the
+[`hyperstart`](https://github.com/hyperhq/hyperstart) VM agent to multiple
+clients on the host.
+
+![High-level Architecture Diagram](../documentation/high-level-overview.png)
```

Comment:
1/ Might be worth showing that, but would rather spend more time on this right now. If you think it's worth the time, could you file an issue so we don't lose the idea?

2/ The diagram was hand drawn using google docs, not generated.